source/beamline/SGM/SGMBypassLadder.cpp

```diff
@@ -0,0 +1,14 @@
+#include "SGMBypassLadder.h"
+
+SGMBypassLadder::SGMBypassLadder(const QString &name, const QString &baseName, QObject *parent) :
+	SGMLadderControl(name, baseName, parent)
+{
+	addValueOption(Slit, "Slit", SGMBYPASSLADDER_SLIT_SETPOINT, SGMBYPASSLADDER_SLIT_MIN, SGMBYPASSLADDER_SLIT_MAX);
```

Comment:
I think I must be missing something, but the first argument looks like it should be a variable of some kind but I don't see it instantiated anywhere.  All the `addValueOption()` methods have it so I'm sure I must be missing something.